### ISSUE TITLE
CSS

### ISSUE THREAD

**Issue by fatima7540** [validator/validator#1531]
```html
<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8">
<title>Ordered list</title>
<style>
p {
    color: blue;
    font-size: 20px;
    width: 200px;
}

h1 {
    color: green;
    font-size: 36px;
    text-align: center;
}
</style>
</head>
<body>
<h1>Anatomy of a CSS Rule</h1>
<h2>Subheading 1</h2>
<p>Now, as I told you, a CSS rule can consist of several declarations. For example, in this case, we're specifying the contents of the paragraph tags in our HTML page should have a particular color, a particular font size, and also a particular width, meaning the paragraph itself, the content, should take up 200 pixels.</p>
<h2>Subheading 2</h2>
<p>Now, as I told you, a CSS rule can consist of several declarations. For example, in this case, we're specifying the contents of the paragraph tags in our HTML page should have a particular color, a particular font size, and also a particular width, meaning the paragraph itself, the content, should take up 200 pixels.</p>
</body>
</html>
```

**Comment by nschonni** [comment]
Pleas don't post spam issues
